networking/multiple_networks/configuring-multi-network-policy.adoc

```diff
@@ -6,14 +6,22 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-As a cluster administrator, you can configure network policy for additional networks.
+As a cluster administrator, you can configure multi-network for additional networks. You can specify multi-network policy for SR-IOV and macvlan additional networks. Macvlan additional networks are fully supported other types of additional networks, such as ipvlan, are not supported.
```

Comment:
I think you need a period here: 

"Macvlan addditional networks are fully supported. Other types of ..."